src/codeflare_sdk/cluster/auth.py

```diff
@@ -101,12 +101,16 @@ def login(self) -> str:
         """
         global config_path
         global api_client
+        odh_ca_path = "/etc/pki/tls/custom-certs/ca-bundle.crt"
```

Comment:
Can we move this into `__init__` and set `self.ca_cert_path` directly? and also allow the default path to be configurable via an environment variable?

Reply:
Those are some great suggestions I'll make those changes